### ISSUE TITLE
about the size of the inserted SEI

### ISSUE THREAD

**Issue by kaikaijiang** [praneet195/FFMPEG_SEI_NALU#2]
Hi Praneet195, really nice work. your work helps me a lot. Thanks a lot.
I have a question related to the insertor code. It looks like the that you set the sei[6] = (uint8_t)128, is the size of the SEI message? As it looks like the inserted SEI message can not be larger 255 Byte.
Thanks in advance.

**Comment by praneet195** [comment]
Yes, that is the max limit. However, you could create multiple sei elements of size 255 and pass them along if needed
